chatmessage/tests.py

```diff
@@ -1,29 +1,30 @@
-from .models import Chatmessage
+from .models import Chatmessage, SenderType
 import pytest
 
-PROFESSIONAL_ID = "professional1"
-CLIENT_ID = "client1"
+
 MESSAGE = "message1"
 
 
 @pytest.fixture
-def chatmessage():
-    return Chatmessage(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, message=MESSAGE)
+def chatmessage(professional, client):
+    return Chatmessage(professional_id=professional,
+                       client_id=client, message=MESSAGE, sender_type=SenderType.Client)
 
```

Comment:
I think you should add it to conftest.py

Reply:
Done